### PR TITLE
esa-snap: add page

### PR DESCRIPTION
**Pull request by sebfoe**
Conflicts with the package manager 'snap' on linux systems are not nice. Any ideas to manage this are highly welcome! Thank you!

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Ref: #9176

### CODE REVIEW

File: pages/linux/esa-snap.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# snap
+
+> SNAP (Sentinel Application Platform) for processing satellite data from the European Space Agency (ESA).
+> More information: <http://step.esa.int/main/download/snap-download/>.
+
+- Show help:
```
Comment (kbdharun) [inline pages/linux/esa-snap.md:6]:
```suggestion
- Display help:
```
Can you move this command to last?

File: pages/linux/esa-snap.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`snap --help`
+
+- Show all updates:
```
Comment (kbdharun) [inline pages/linux/esa-snap.md:10]:
```suggestion
- Display all updates:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

File: pages/linux/esa-snap.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+> SNAP (Sentinel Application Platform) for processing satellite data from the European Space Agency (ESA).
+> More information: <http://step.esa.int/main/download/snap-download/>.
+
+- Display help:
+
+`snap --help`
+
+- Display all updates:
+
+`snap --nosplash --nogui --modules --list --refresh`
```
Comment (navarroaxel) [inline pages/linux/esa-snap.md:12]: Move help to the bottom
```suggestion
- Display all updates:

`snap --nosplash --nogui --modules --list --refresh`

- Display help:

`snap --help`
```

File: pages/linux/esa-snap.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# snap
+
+> SNAP (Sentinel Application Platform) for processing satellite data from the European Space Agency (ESA).
```
Comment (navarroaxel) [inline pages/linux/esa-snap.md:3]: Perhaps this better
```suggestion
> Sentinel Application Platform (SNAP) for processing satellite data from the European Space Agency (ESA).
```

**Review by EmilySeville7cfg — APPROVED** [review]
PR is approved because possible edits are already mentioned and should be applied (there is nothing I can suggest here).

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**